src/pages/dev/amplifier/chain-integration/governance-proposals.mdx

```diff
@@ -0,0 +1,341 @@
+
+## Community Forum introduction
+
+You will need to introduce your chain to the community in order for Axelar token holders to vote for your proposals. This should be done on the governance section of the [Axelar Community Forum](https://community.axelar.network/c/governance2/33). Make a community introduction post that precedes and can be referenced by any on-chain proposals. It should include the following information:
+
+- Background
+- Governance details
+- Pre-deployed source chain gateway address
+- Plan for funding rewards pools
+
+All of your on-chain proposals will refer back to this introduction, so make sure to be thorough. You should stay present and engage with any questions or comments during the seven-day feedback period, which can include public comments as well as a security council review. When the feedback period is over, submit your on-chain governance proposals to finish the integration of your chain.
+
+<Callout>
+Examples were created with contracts on the `ethereum-sepolia` chain. Please refer to [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json) for contracts associated with other chains.
+</Callout>
+
+## Proposal to instantiate the verifier contract
+
+<tabs>
+
+<tab-item title="EVM integrators">
+
+If you are integrating an EVM chain, you can submit a proposal to instantiate the pre-deployed voting verifier contract with the [proper instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/deab56a75209a158c082cbf9b77caf500eb4ec7c/contracts/voting-verifier/src/msg.rs#L9) using its existing `code_id` (`500`):
+
+```bash
+export CODE_ID=500
+export NETWORK_GOV_ADDRESS="axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj"
+export RPC="http://devnet-amplifier.axelar.dev:26657"
+```
+
+```bash
+axelard tx gov submit-proposal instantiate-contract $CODE_ID \
+'{
+    "governance_address": "axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj",
+    "service_registry_address": "axelar1c9fkszt5lq34vvvlat3fxj6yv7ejtqapz04e97vtc9m5z9cwnamq8zjlhz",
+    "service_name":"validators",
+    "source_gateway_address":"0xeE9E463Fd8bE9AF266e1B143C27F7D42648f4006",
+    "voting_threshold":["1","1"],
+    "block_expiry":10,
+    "confirmation_height":1,
+    "source_chain":"ethereum-sepolia",
+"rewards_address":"axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l",
+    "msg_id_format":"hex_tx_hash_and_event_index"
+}' \
+--title="Instantiate verifier contract for [chain name]" \
+--description="Instantiate verifier contract for [chain name]" \
+--run-as $NETWORK_GOV_ADDRESS \
+--label "[chain name]" \
+--deposit=2000000000uamplifier \
+--keyring-backend test \
+--from wallet \
+--gas auto --gas-adjustment 1.5 --gas-prices 0.007uamplifier \
+--chain-id=devnet-amplifier \
+--no-admin \
```

Comment:
Do we want no-admin, or should this be an address or the governance address? cc @cgorenflo

Reply:
Tested with `--no-admin` because Verifier contract does not have an admin address: [https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json#L306](https://www.google.com/url?q=https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json%23L306&sa=D&source=docs&ust=1721508858049865&usg=AOvVaw1WQeD_IAxMy-wcbNasCIkE)

cc @canhtrinh